.gitlab-ci.yml

```diff
@@ -141,3 +137,22 @@ buildah:
         - .gitlab-ci.yml
         - assets/**/*
     - if: '$CI_PIPELINE_SOURCE == "schedule" && $BUILD_CONTAINER == "1"'
+
+buildah_base:
+  extends:
+    - .parallel_distro
+    - .buildah
+  before_script:
+    - IMAGE_NAME="${REGISTRY_USER}/${DISTRO}:${VERSION}"
+    - buildah login -u ${REGISTRY_USER} -p ${REGISTRY_TOKEN} docker.io
+
+buildah_custom:
```

Comment:
What is .buildah_custom for?

Reply:
To not launch the gitlab tests on all the images. Normally it is already done by GitHub.
It's just to build more images and change the name of images (Bitnami/minideb --> minideb)